humanprotocol/human-protocol
# [Job Launcher] Fix password validation regex

## Description
Used the same regular expression in the backend that we use in the [frontend](https://github.com/humanprotocol/human-protocol/blob/4a1d2830fa59277847877b80a537c622ac8669fc/packages/apps/job-launcher/client/src/components/Auth/schema.ts#L15)

## How to test
```
curl -X 'POST' \
  'http://localhost:5001/auth/signup' \
  -H 'accept: */*' \
  -H 'Content-Type: application/json' \
  -d '{
  "password": "Developer123_",
  "confirm": "Developer123_",
  "email": "test@hmt.ai"
}'
```

## Related issues
[Job Launcher - password validation #1363](https://github.com/humanprotocol/human-protocol/issues/1363)
